src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/DataFrameTests.cs

```diff
@@ -404,6 +404,22 @@ public void TestSignaturesV2_3_X()
             Assert.IsType<RelationalGroupedDataset>(_df.GroupBy(_df["age"]));
             Assert.IsType<RelationalGroupedDataset>(_df.GroupBy(_df["age"], _df["name"]));
 
+            Assert.IsType<DataFrame>(_df.GroupBy("name").Mean("age"));

```

Comment:
@imback82  should we have a separate test suite for RelationalGroupedDataset or are these fine here ?

Reply:
fine for now.